contracts/protocol/AirnodeProtocol.sol

```diff
@@ -20,10 +20,10 @@ import "./interfaces/IAirnodeProtocol.sol";
 /// StorageUtils, SponsorshipUtils and WithdrawalUtils also implement some
 /// auxiliary functionality for PSP.
 contract AirnodeProtocol is
+    ExtendedSelfMulticall,
```

Comment:
tryMultical()l at AirnodeProtocol allows batch fulfillment of RRP requests (though that means fulfillments sharing gas limit so it's not very straightforward to implement)

Reply:
Yeah. I suppose it would also have to be limited to requests that specify `_gasLimit` (which doesn't exist) and we would need to add these up for the batch fulfillment